roles/kubevirt_web_ui/tasks/install.yml

```diff
@@ -0,0 +1,131 @@
+---
+- name: Check if kubevirt-web-ui project exists
+  shell: "{{ openshift_client_binary }} projects -q | grep -w kubevirt-web-ui | awk '{ print $1 }'"
+  register: ns
+
+- name: Create kubevirt-web-ui project
+  shell: "{{ openshift_client_binary }} new-project kubevirt-web-ui"
+  when: ns.stdout != "kubevirt-web-ui"
+
+- name: Make temp directory for console templates
+  command: mktemp -d /tmp/console-ansible-XXXXXX
```

Comment:
ansible may have a built in command

Reply:
Found this one: https://docs.ansible.com/ansible/2.5/modules/tempfile_module.html

Would be simpler change than with the jinja2 template, but again: the role will be harder to compare with its openshift-ansible counterpart.

Can we keep it as it is? From semantical POV, there would be no difference.

Reply:
will leave as is